resources/function_help/json/replace

```diff
@@ -1,10 +1,13 @@
 {
   "name": "replace",
   "type": "function",
-  "description": "Returns a string with the the supplied string replaced.",
+  "description": "Returns a string with the supplied string or array of strings replaced by a string or an array of strings. The function can also work with two arguments, whereas the second argument is a map value used to replace keys with associated string values.",
```

Comment:
Have a look at the "length" function help - that's a good example how variants such be handled for functions.